receiver/purefareceiver/config.go

```diff
@@ -42,7 +42,7 @@ type Config struct {
 	Env string `mapstructure:"env"`
 
 	// Fa_array_name represents the display name that is appended to the received metrics, as the `host` label if not provided by OpenMetrics output, and to the `fa_array_name` label always.
-	Fa_array_name string `mapstructure:"fa_array_name"`
+	ArrayName string `mapstructure:"fa_array_name"`
```

Comment:
You'll need to apply the rename elsewhere in this PR as well

Reply:
This should be renamed where it needs to be. It was updated in config and receiver and tests